graph/test/test_srt.c

```diff
@@ -12,6 +12,7 @@ typedef struct {
 
 typedef RAGE_OR_ERROR(rage_TestElem) rage_NewTestElem;
 
+// FIXME: this is getting out of hand
```

Comment:
What is getting out of hand? The stuff that has to change because of the new struct?

Reply:
The amount of stuff going on in this function generally was quite a bit already, with the extra level to keep track of it is starting to tip over to the unmanagable side.